src/UsgsAstroLsSensorModel.cpp

```diff
@@ -479,8 +479,10 @@ csm::ImageCoord UsgsAstroLsSensorModel::groundToImage(
    else {
       --approxNextPoint.line;
    }
-   csm::EcefCoord approxIntersect = imageToGround(approxPoint, 0);
-   csm::EcefCoord approxNextIntersect = imageToGround(approxNextPoint, 0);
+   double height, aPrec;
+   computeElevation(ground_pt.x, ground_pt.y, ground_pt.z, height, aPrec, desired_precision);
+   csm::EcefCoord approxIntersect = imageToGround(approxPoint, height);
+   csm::EcefCoord approxNextIntersect = imageToGround(approxNextPoint, height);

```

Comment:
This is only tangentially related to this PR, but when the computeLinearApproximation is called above, where/how is the _linear variable being set to not just default to the center of the image?

I am asking because, are we starting close for the secant method to be able to not hit a potential local 0?

Reply:
replaceModelState attempts to set the linear approximation. We don't actually use this linear approximation as an input into the secant method search. It's possible we should to save an iteration or two.

Reply:
The main use for the approximate point is to convert the desiredPrecision from ground distance to image distance.